cms/templates/cms/home_page.html

```diff
@@ -118,79 +118,74 @@ <h5 class="upper-case">Save Money &amp; Time</h5>
           <blockquote class="quote-text">“Inverted admissions has the potential to disrupt traditional modes of access to higher education… We’re democratizing access to a master’s program for learners worldwide.”</blockquote>
           <p class="quote-author">-- Sanjay Sarma, Vice President of Open Learning, MIT  </p>
         </div>
-      </div>
+      </section>
 
-      <div class="row current-programs-section">
-        <div class="col-md-12">
-          <div class="row">
-            <h3>Current MITx MicroMasters Programs</h3>
-            {% for program in programs %}
-              <div class="col-md-4">
-                {% if program.programpage %}
-                  {% if program.programpage.external_program_page_url %}
-                    <a href="{{ program.programpage.external_program_page_url }}" class="program-link">
-                  {% else %}
-                    <a href="{{ program.programpage.url }}" class="program-link">
-                  {% endif %}
+      <section class="row current-programs-section">
+        <h3>Current MITx MicroMasters Programs</h3>
+        <ul class="current-programs-list">
+        {% for program in programs %}
+          <li class="program-card col-md-4">
```

Comment:
I thought I was going to use it for styling, and then it turned out that I didn't have to. I've removed it from this pull request.